guacamole/src/main/webapp/translations/cs.json

```diff
@@ -42,8 +42,9 @@
 
         "INFO_ACTIVE_USER_COUNT" : "V současné době používá {USERS} {USERS, plural, one{user} other{users}}.",
 
-        "TEXT_ANONYMOUS_USER"   : "Anonym",
-        "TEXT_HISTORY_DURATION" : "{VALUE} {UNIT, select, second{{VALUE, plural, one{second} other{seconds}}} minute{{VALUE, plural, one{minute} other{minutes}}} hour{{VALUE, plural, one{hour} other{hours}}} day{{VALUE, plural, one{day} other{days}}} other{}}"
+        "TEXT_ANONYMOUS_USER"   : "Neznámý uživatel",
+        "TEXT_HISTORY_DURATION" : "{VALUE} {UNIT, select, second{{VALUE, plural, one{second} other{seconds}}} minute{{VALUE, plural, one{minute} other{minutes}}} hour{{VALUE, plural, one{hour} other{hours}}} day{{VALUE, plural, one{day} other{days}}} other{}}",
```

Comment:
This will result in the English words for these units and the English plurals being used (second/seconds, minute/minutes, etc.). Is this intentional?

Reply:
Offf, sorry, missed it! Fixed! :-) thanks!